python/src/ExpectationSimulationAlgorithm_doc.i.in

```diff
@@ -11,70 +11,268 @@ X : :class:`~openturns.RandomVector`
-ExpectationSimulationResult
+ExpectationSimulationResult, SimulationAlgorithm
 
 Examples
 --------
+
+In the following example, we perform at most :math:`8 \times 1000 = 8000` evaluations 
+of the model. 
+However, the algorithm may stop earlier if the coefficient of variation 
+of the sample mean falls below the threshold. 
+
 >>> import openturns as ot
 >>> ot.RandomGenerator.SetSeed(0)
 >>> # Create a composite random vector
 >>> model = ot.SymbolicFunction(['E', 'F', 'L', 'I'], ['-F*L^3/(3*E*I)'])
->>> distribution = ot.Normal([50.0, 1.0, 10.0, 5.0], [1.0]*4, ot.IdentityMatrix(4))
+>>> distribution = ot.Normal([50.0, 1.0, 10.0, 5.0], [1.0] * 4, ot.IdentityMatrix(4))
 >>> vect = ot.RandomVector(distribution)
 >>> X = ot.CompositeRandomVector(model, vect)
+>>> algo = ot.ExpectationSimulationAlgorithm(X)
+>>> algo.setMaximumOuterSampling(10000)
+>>> algo.setBlockSize(8)  # The number of cores we usually have.
+>>> algo.setMaximumCoefficientOfVariation(0.01)  # 1% C.O.V.
+>>> algo.run()
+>>> result = algo.getResult()
+>>> expectation = result.getExpectationEstimate()
+>>> print(expectation)
+[-1.41067]
+>>> expectationDistribution = result.getExpectationDistribution()
+
+The following code prints the outer sample (i.e. the number of outer iterations 
+of the algorithm) and the sample size. 
+We see that the algorithm stops based on the accuracy criterion. 
+
+>>> outer_sampling = result.getOuterSampling()
+>>> print('outer_sampling=', outer_sampling)
+outer_sampling= 1662
+>>> block_size = result.getBlockSize()
+>>> sample_size = outer_sampling * block_size
+>>> print('sample_size=', sample_size)
+sample_size= 13296
+
+In the following example, we disable the coefficient of variation criterion. 
+Therefore, the only criteria is based on the number of iterations. 
+
 >>> algo = ot.ExpectationSimulationAlgorithm(X)
 >>> algo.setMaximumOuterSampling(1000)
 >>> algo.setBlockSize(1)
 >>> algo.setCoefficientOfVariationCriterionType('NONE')
 >>> algo.run()
 >>> result = algo.getResult()
+>>> outer_sampling = result.getOuterSampling()
+>>> print('outer_sampling=', outer_sampling)
+outer_sampling= 1000
+
+In the following example, we compute the mean of a dimension 4 random vector. 
+We configure the stopping criterion in order to stop when the componentwise 
+standard deviation is low than a threshold. 
+We must disable the coefficient of variation stopping criterion, otherwise 
+the algorithm stops on this criterion first, in this example. 
+The comparison with the exact mean is satisfactory, given the relatively 
+small sample size.
+
+>>> from openturns.usecases import cantilever_beam as cantilever_beam
+>>> cb = cantilever_beam.CantileverBeam()
+>>> sigma = cb.distribution.getStandardDeviation()
+>>> print('sigma=', sigma)
+sigma= [1.73582e+09,30,0.0288675,7.10585e-09]
+>>> componenwise_max_sigma = sigma / 32.0
+>>> print('componenwise_max_sigma=', componenwise_max_sigma)
+componenwise_max_sigma= [5.42445e+07,0.9375,0.00090211,2.22058e-10]
```

Comment:
```suggestion
>>> componentwise_max_sigma = sigma / 32.0
>>> print('componentwise_max_sigma=', componentwise_max_sigma)
componentwise_max_sigma= [5.42445e+07,0.9375,0.00090211,2.22058e-10]
```

Reply:
I cannot see the actual suggestion: there is no difference, isn'it?

Reply:
Oh; the missing "t".